config/checkstyle/suppressions.xml

```diff
@@ -59,6 +59,10 @@
     <suppress checks="JavadocPackage"
               files=".*[/\\]subprojects[/\\]ide-native[/\\]src[/\\]main[/\\]java[/\\]org[/\\]gradle[/\\]plugins[/\\]ide[/\\]api[/\\][^/\\]+"/>
 
+    <!-- These packages are duplicated in platform-native from language-native, don't require a package-info.java in each place -->
```

Comment:
do we need this? looks like a left over some some refactoring that was reverted.

Reply:
I can't remove this as `SwiftSourceCompatibility` enum is used in the Swift compiler class so it needs to be in the `platform-native` subproject. It doesn't make sense to move it in another package either.